src/components/Map/SearchPane/Drawer/DrawerContent/EntityDetails/EntityComponents/EntityInfo/EntityInfo.tsx

```diff
@@ -32,24 +42,28 @@ const EntityInfo: React.FC<PageProps> = (props) => {
     useEffect(() => {
         if(props.emissionsData && props.entitiesByBusinessType.length === 0) {
             getEntityDetails();
-        }
+        };
+        checkIsPortrait();
     }, []);
     return (
         <div>
             {entityDetails ?
                 <div>
-                    <img 
-                        className="entity-img"
-                        src={entityDetails.img}
-                    />
-                    <div style={{ display: 'flex', flexDirection: 'row', alignItems: 'center', justifyContent: 'space-between' }}>
-                        <div style={{ display: 'flex', flexDirection: 'column' }}>
-                            <IonText className="ion-text-capitalize ion-text-left entity-title">{entityDetails.name}</IonText>
-                            <IonText className="ion-text-capitalize entity-address">{entityDetails.address}</IonText>
+                    <div style={{ display: isPortrait ? 'initial' : 'flex', flexDirection: 'row' }}>
```

Comment:
Can you use the CSS `orientation` selector for this rather than your own TS `isPortrait`? e.g. https://developer.mozilla.org/en-US/docs/Web/CSS/@media/orientation
This would keep the style logic in the CSS rather than inline in the TS